lib/Dropdown/DropdownTrigger.js

```diff
@@ -48,6 +49,10 @@ class DropdownTrigger extends Component {
 
     const buttonTypes = useSelect ? types.concat('outline') : types;
 
+    if (renderChildrenOnly) {
+      return children;
```

Comment:
Nice! One q though do we need to wrap this in the div so we can use the ref? (im not sure what the ref is used for without digging into it! so might be completely redundant)

Reply:
Yes we do, good catch 👍

Reply:
@AmeeMorris @DavidHollins6 

This was a great catch as it turns out! My implementation was totally wrong.

Not only do we need to wrap the `children` in a `div` to attach the `ref`, I also need to pass down `this.handleClick` from this function to `children`. This function does all the auto positioning logic.

Previously I was just using `setShowContent` that is passed out of the overarching `Dropdown` component, which does not take into account any autopositioning.

I have updated this so the component now checks to see if its `children` is a function, and then passes the correct `this.handleClick` function to it. Plus the `div` wrapping with the `ref`.

Commit is here! https://github.com/gathercontent/gather-ui/pull/721/commits/05d21fc239d43a3f20e5b5bd76e05a81179f94b7